opendevin/mock/listen.py

```diff
@@ -42,5 +42,9 @@ def read_llm_agents():
         "PlannerAgent",
     ]
 
+@app.get("default-model")
```

Comment:
shouldn't there be a slash at that start of this string?

Reply:
Yes, my bad.